From: Lasse Collin <lasse.collin@tukaani.org>
Date: Sun, 1 Nov 2020 18:49:37 +0200
Subject: xz: Protect the ellipsis (...) on the man page with \&.

This does it only when ... appears outside macro calls.

Thanks to Bjarni Ingi Gislason.
---
 src/xz/xz.1 | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/xz/xz.1 b/src/xz/xz.1
index c8cb153..f659be0 100644
--- a/src/xz/xz.1
+++ b/src/xz/xz.1
@@ -1088,7 +1088,7 @@ A custom filter chain allows specifying
 the compression settings in detail instead of relying on
 the settings associated to the presets.
 When a custom filter chain is specified,
-preset options (\fB\-0\fR ... \fB\-9\fR and \fB\-\-extreme\fR)
+preset options (\fB\-0\fR \&...\& \fB\-9\fR and \fB\-\-extreme\fR)
 earlier on the command line are forgotten.
 If a preset option is specified
 after one or more custom filter chain options,
@@ -1172,7 +1172,7 @@ The integer can be from
 .B 0
 to
 .BR 9 ,
-matching the command line options \fB\-0\fR ... \fB\-9\fR.
+matching the command line options \fB\-0\fR \&...\& \fB\-9\fR.
 The only supported modifier is currently
 .BR e ,
 which matches
